packages/core/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@pixi/constants": "6.0.0-rc.3",
+    "@pixi/events": "^6.0.0-rc.3",
```

Comment:
Again, to start, I'd like to remove the EventSystem from the renderer. Users can `renderer.addSystem(EventSystem, 'events')` until I get the static installing of systems working with Renderer.

Reply:
Reverted 👍